js/.eslintrc.js

```diff
@@ -154,6 +154,18 @@ module.exports = {
     files: ['web/lib/wasm/binding/**/*.ts'], rules: {
       '@typescript-eslint/naming-convention': 'off'
     }
+  }, {
```

Comment:
there are several `.js` source file under react native folder. do you want to enable eslint for `.js` there? I disabled `.js` because existing folders contains almost no `.js` source files.

Reply:
babel.config.js is an auto-generated file by react native bob builder. other js files are for build and package. all others are ts or tsx.

Reply:
change prepack to typescript.